Author: Pino Toscano <toscano.pino@tiscali.it>
Description: Remove extra include on GNU/kFreeBSD.
 Due to Debian #704598, GNU/kFreeBSD' sys/mount.h cannot be used in
 C90 C sources, such as plat_freebsd.c when compiled with default CFLAGS
 provided with kdelibs.
 It seems redundant, so just remove it.
Forwarded: no
Last-Update: 2014-04-08

--- a/wmlib/plat_freebsd.c
+++ b/wmlib/plat_freebsd.c
@@ -36,7 +36,6 @@
 #include <unistd.h>
 #include <sys/types.h>
 #include <sys/param.h>
-#include <sys/mount.h>
 #include <sys/stat.h>
 
 #include "include/wm_config.h"
